### PR TITLE
Employes page

### PR DESCRIPTION
**Pull request by Manu92915**

### CODE REVIEW

File: src/Web/Client/Common/Extensions/MenuItemExtension.cs [hunk 1]
```diff
@@ -15,4 +16,9 @@ public static MenuItem ToMenuItem(this GetTag tag)
     {
         return new MenuItem(tag.Id, tag.Name, "", $"/settings/tags/{tag.Id}");
     }
+
+    public static MenuItem ToMenuItem(this GetEmployee employee)
+    {
+        return new MenuItem(employee.Id, employee.FullName, "", $"/settings/tags/{employee.Id}");
```
Comment (SidiqovAbbos) [inline src/Web/Client/Common/Extensions/MenuItemExtension.cs]: `settings/tags/` -> `/employees/`

File: src/Web/Client/Pages/Employees.razor [hunk 2]
```diff
@@ -1,18 +1,99 @@
...
+            _getEmployee = await EmployeesService.GetEmployeeById(Id);
+        }
+        _loading = false;
+        StateHasChanged();
+    }
+
+    private async void CreateOrUpdate()
     {
-        currentCount++;
+        if (Id is not null)
```
Comment (SidiqovAbbos) [inline src/Web/Client/Pages/Employees.razor]: if (Id is null) {
   create employee
}

File: src/Web/Client/Common/Extensions/MenuItemExtension.cs (hunk 1 above)
Comment (Manu92915) [reply src/Web/Client/Common/Extensions/MenuItemExtension.cs]: Fixed

File: src/Web/Client/Pages/Employees.razor (hunk 2 above)
Comment (Manu92915) [reply src/Web/Client/Pages/Employees.razor]: Changed

**Review by SidiqovAbbos — APPROVED** [review]

**Pull request merged**